vkoves/electrify-chicago
# [DRAFT] Implement Docker

# Description

We want to streamline our setup process for our developers and improve our version control by setting up Docker for electrify-chicago. Changes made:

1. Added Dockerfile
2. Added docker-compose up.yml
3. Added .dockerignore file

Fixes #84

# Testing Instructions

1. Run `docker-compose up`
2. Confirm image is created and site is visible at localhost
3. I will add better testing instructions when this is actually ready to be tested 

# Checklist:
